app/Transformers/TracksTransformer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Transformers;
+
+use App\Track;
+use League\Fractal\TransformerAbstract;
+
+class TracksTransformer extends TransformerAbstract
+{
+    /**
+     * @param Track $track
+     * @return array
+     */
+    public function transform(Track $track)
+    {
+        return [
+            'id' => $track->id,
+            'slug' => $track->slug,
+            'album_id' => $track->album_id,
+            'name' => $track->name,
+            'file' => $track->file_path,
```

Comment:
so currently we have two options, either upload a mp3 or specify a url
file_path is the path to the uploaded file
mp3_link is the link to the external mp3 file

i dont know if we want to change this in the database level and change the names